src/components/ebay-star-rating/star-rating.stories.js

```diff
@@ -24,12 +26,37 @@ export default {
         },
     },
 
-    argTypes: {},
+    argTypes: {
+        value: {
+            control: { type: 'text' },
```

Comment:
Maybe the value should be controlled with a dropdown so only valid options can be used. 
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/1675667/164504822-3cd07198-7720-4017-93f6-7e0028e9d465.png">

Reply:
Good idea